src/pyOpenMS/scripts/get_build_requires.py

```
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
```

Ruff (0.14.7)

Shebang is present but file is not executable

(EXE001)